### PR TITLE
Fix/org discovery doc issue

### PR DESCRIPTION
**Pull request by BimsaraBodaragama**
- An unnecessary “3” in the API URL.

- [Remove Unnecessary Content in API URL for Email Domain Based Organization Discovery #25151](https://github.com/wso2-enterprise/asgardeo-product/issues/25151)

[25151](https://github.com/wso2-enterprise/asgardeo-product/issues/25151): Removed the unnecessary “3” from the API URL format to ensure the URL is correct and functional.

### CODE REVIEW

File: en/asgardeo/docs/guides/organization-management/email-domain-based-organization-discovery.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 {% set host_name = "api.asgardeo.io" %}
 {% set organization_path_param = "/t/<organization_name>"  %}
+{% set parent_organization_path_param = "/t/<parent_organization_name>"  %}
```
Comment (AnuradhaSK) [inline en/asgardeo/docs/guides/organization-management/email-domain-based-organization-discovery.md:3]:
```suggestion
{% set root_organization_name = "/t/<root_organization_name>"  %}
```
to  be consistent with other places

File: en/identity-server/7.0.0/docs/guides/organization-management/email-domain-based-organization-discovery.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
 {% set host_name = "localhost:9443" %}
 {% set organization_path_param = ""  %}
+{% set parent_organization_path_param = ""  %}
```
Comment (AnuradhaSK) [inline en/identity-server/7.0.0/docs/guides/organization-management/email-domain-based-organization-discovery.md:3]:
```suggestion
{% set root_organization_name = ""  %}
```

File: en/includes/guides/organization-management/email-domain-based-organization-discovery.md [hunk 3]
```diff
@@ -127,7 +127,7 @@ You need to:
 2. Use the obtained access token to execute the following cURL.
 
     ``` curl
-    curl --location --request PUT 'https://{{ host_name }}{{ organization_path_param }}/api/server/v1/organizations/<organization-id>/discovery' \
+    curl --location --request PUT 'https://{{ host_name }}{{ parent_organization_path_param }}/api/server/v1/organizations/<organization-id>/discovery' \
```
Comment (AnuradhaSK) [inline en/includes/guides/organization-management/email-domain-based-organization-discovery.md:130]:
```suggestion
    curl --location --request PUT 'https://{{ host_name }}{{ root_organization_name }}/api/server/v1/organizations/<organization-id>/discovery' \
```

File: en/includes/guides/organization-management/email-domain-based-organization-discovery.md [hunk 4]
```diff
@@ -89,7 +89,7 @@ You need to:
 2. Use the obtained access token to execute the following cURL.
 
     ``` curl
-    curl --location --request POST 'https://{{ host_name }}{{ organization_path_param }}3/api/server/v1/organizations/discovery' \
+    curl --location --request POST 'https://{{ host_name }}{{ organization_path_param }}/api/server/v1/organizations/discovery' \
```
Comment (AnuradhaSK) [inline en/includes/guides/organization-management/email-domain-based-organization-discovery.md:92]:
```suggestion
    curl --location --request POST 'https://{{ host_name }}{{ root_organization_name }}/api/server/v1/organizations/discovery' \
```

File: en/asgardeo/docs/guides/organization-management/email-domain-based-organization-discovery.md (hunk 1 above)
Comment (BimsaraBodaragama) [reply en/asgardeo/docs/guides/organization-management/email-domain-based-organization-discovery.md]: Done.

File: en/identity-server/7.0.0/docs/guides/organization-management/email-domain-based-organization-discovery.md (hunk 2 above)
Comment (BimsaraBodaragama) [reply en/identity-server/7.0.0/docs/guides/organization-management/email-domain-based-organization-discovery.md]: Done.

File: en/includes/guides/organization-management/email-domain-based-organization-discovery.md (hunk 3 above)
Comment (BimsaraBodaragama) [reply en/includes/guides/organization-management/email-domain-based-organization-discovery.md]: Done.

File: en/includes/guides/organization-management/email-domain-based-organization-discovery.md (hunk 4 above)
Comment (BimsaraBodaragama) [reply en/includes/guides/organization-management/email-domain-based-organization-discovery.md]: Done.

**Comment by AnuradhaSK** [comment]
Similar to identity-server/7.0.0 path fix should go to identity-server/next path's docs as well

**Review by AnuradhaSK — APPROVED** [review]

**Pull request merged**